packages/angular_devkit/build_angular/src/tools/esbuild/commonjs-checker.ts

```diff
@@ -83,7 +83,7 @@ export function checkCommonJSModules(
         const request = imported.original;
 
         let notAllowed = true;
-        if (allowedRequests.has(request)) {
+        if (allowedRequests.has('*') || allowedRequests.has(request)) {
```

Comment:
In the case `allowedRequests.has('*')` is trurthy we can skip all the logic in this plugin. Consider doing this change at the beginning.